test-integration/helpers/ui/user.coffee

```diff
@@ -104,8 +104,7 @@ class User extends TestHelper
       hider = document.createElement('style')
       hider.textContent = '''
         .navbar-fixed-bottom, .navbar-fixed-top,
-        .pinned-on .pinned-header,
-        .pinned-footer {
+        .pinned-on .pinned-header {
```

Comment:
it affected whether the continue button could be found in completing student tasks I think.  I dunno, i think we should manipulate the UI as little as possible for testing